core/src/main/scala/za/co/absa/spline/harvester/converter/AttributeIdResolver.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester.converter
+
+import org.apache.spark.sql.catalyst.expressions.ExprId
+
+import scala.collection.mutable
+
+class AttributeIdResolver {
```

Comment:
Is this class actually needed? All it does is wrapping a mutable map. The usage of the composite key also seems to be unnecessary, as we a priori are dealing with a logical execution plan created on the same JVM.